webhallen-sum.user.js

```diff
@@ -311,23 +311,26 @@ GM_addStyle('@import url("https://unpkg.com/charts.css/dist/charts.min.css");');
 
     function getStoreStats(orders) {
         const storePurchases = orders.reduce((stores, order) => {
-            const storeName = order.store?.name ?? 'N/A';
+            let storeName = order.store?.name;
+            if (storeName === undefined) {
+                storeName = order.shippingMethod.name;
```

Comment:
Is this desired behavior? I.e. having a store called "Postpaket"? Maybe we could create a separate chart for shipping methods to make it easier to compare _just_ shippingmethods or _just_ stores?

Reply:
Good point, I just wanted to not see "N/A" and without much thought I appended the shipping method.
This can of course be moved/removed

Reply:
In that case I think `if (!storeName) return stores` would work here since it would skip counting orders that dont have a store.